src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,26 @@
+import { useTodos } from '../../context/TodosContext';
+import { TodoItem } from '../TodoItem';
+
+export const TodoList: React.FC = () => {
+  const { todos, filterStatus } = useTodos();
+
+  const filteredTodos = todos.filter(todo => {
+    switch (filterStatus) {
+      case 'Active':
```

Comment:
Use Enum values instead of the hardcoded values

Reply:
I use keys from Enum here 

<img width="201" alt="Screenshot 2024-03-11 140334" src="https://github.com/mate-academy/react_todo-app/assets/130569769/844b78df-1970-4c02-a733-689062251959">

<img width="266" alt="Screenshot 2024-03-11 140411" src="https://github.com/mate-academy/react_todo-app/assets/130569769/eb6d4fab-8754-4ec6-9ce5-cf0103f0f599">

and it's not  hardcoded because when I type key that doesn't exist in my Status Emun I get an error

<img width="341" alt="Screenshot 2024-03-11 140702" src="https://github.com/mate-academy/react_todo-app/assets/130569769/ae85a46b-649a-4794-8fd2-9dc3dba40989">